console/src/main/java/org/georchestra/console/ws/AreaController.java

```diff
@@ -0,0 +1,82 @@
+package org.georchestra.console.ws;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Controller;
+import org.springframework.util.StreamUtils;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+/**
+ * A simple controller to serve an area.json file from your datadir.
+ */
+@Controller
+public class AreaController {
+
+    @Value("${AreasUrl:https://www.geo2france.fr/public/communes_zones_competence.geojson}")
+    private String areasUrl;
+
+    @Value("${georchestra.datadir:/etc/georchestra}")
+    private String datadir;
+
+    /**
+     * serve a json file put in the datadir or redirect to a url if AreaUrl in the
+     * config is set to a url.
+     * 
+     * @return json or rediurect to the resource if area is an URL
+     * @throws IOException
+     */
+    @GetMapping(value = "/area.json", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
```

Comment:
`/public/area.json` ?
So it falls into https://github.com/georchestra/datadir/blob/0010f8d609788267451fa12dd0583caf1f5598a0/security-proxy/security-mappings.xml#L11

Reply:
I thought we were now striving to not depend on the SP for securing the endpoint and let the app do it ? I didn't secure this endpoint (no security check), but I can change the ressource to `/console/public/area.geosjon`.

Reply:
> I thought we were now striving to not depend on the SP for securing the endpoint and let the app do it ?

You're right, but it's not done yet. In addition, it helps keeping a kind of logic with existing services.